### PR TITLE
Return `schema.table` for table name columns in dolt diff/status tables and functions for doltgres

### PR DESCRIPTION
**Pull request by tbantle22**

### CODE REVIEW

**Comment by coffeegoddd** [comment]
@tbantle22 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>2362514</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>2362514</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

File: go/libraries/doltcore/sqle/dprocedures/dolt_conflicts_resolve.go [hunk 1]
```diff
@@ -362,9 +362,9 @@ func ResolveSchemaConflicts(ctx *sql.Context, ddb *doltdb.DoltDB, ws *doltdb.Wor
 
 	// TODO: schema names
 	tblSet := set.NewStrSet(tables)
```
Comment (zachmu) [inline go/libraries/doltcore/sqle/dprocedures/dolt_conflicts_resolve.go:364]: Probably could use a doltdb.TableNameSet here and get rid of the .Name use below? Not positive though

File: go/libraries/doltcore/doltdb/workingset.go [hunk 2]
```diff
@@ -618,7 +621,7 @@ func (ws *WorkingSet) writeValues(ctx context.Context, db *DoltDB, meta *datas.W
 			return nil, err
 		}
 
-		mergeState, err = datas.NewMergeState(ctx, db.vrw, preMergeWorking, dCommit, ws.mergeState.commitSpecStr, ws.mergeState.unmergableTables, ws.mergeState.isCherryPick)
+		mergeState, err = datas.NewMergeState(ctx, db.vrw, preMergeWorking, dCommit, ws.mergeState.commitSpecStr, FlattenTableNames(ws.mergeState.unmergableTables), ws.mergeState.isCherryPick)
```
Comment (zachmu) [inline go/libraries/doltcore/doltdb/workingset.go:624]: TODO is probably here: we need to serialize the full TableName, which is slightly more challenging

**Review by zachmu — APPROVED** [review]
LGTM

**Comment by coffeegoddd** [comment]
@tbantle22 __DOLT__
 <table>
<TR><TH>comparing_percentages</TH>
</TR>
<TR><TD>100.000000 to 100.000000</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>result</TH>
<TH>total</TH>
</TR>
<TR><TD>4587cad</TD>
<TD>ok</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>version</TH>
<TH>total_tests</TH>
</TR>
<TR><TD>4587cad</TD>
<TD>5937457</TD>
</TR>
</table>
<table>
<TR><TH>correctness_percentage</TH>
</TR>
<TR><TD>100.0</TD>
</TR>
</table>

**Pull request merged**